custom_components/mqtt_vacuum_camera/camera.py

```diff
@@ -401,10 +403,13 @@
     async def _process_parsed_json(self, test_mode: bool = False):
         """Process the parsed JSON data and return the generated image."""
         if test_mode:
-            parsed_json = await self.load_test_json(
-                "custom_components/mqtt_vacuum_camera/snapshots/test.json"
+            _LOGGER.debug("Camera Test Mode Active...")
+            parsed_json = await async_load_file(
+                file_to_load="custom_components/mqtt_vacuum_camera/snapshots/test.json",
+                is_json=True,
             )
-            return parsed_json
+            self._should_poll = False
+            return parsed_json, test_mode
```

Comment:
@coderabbitai await async_load_file in files_operations is correctly handling the errors in case the file isn't present or there is a decode error this try isn't necessary in my option.